modules/managing-namespace-auto-pruning-policies.adoc

```diff
@@ -29,10 +29,100 @@ FEATURE_AUTO_PRUNE: true
 # ...
 ----
 
+[id="creating-registry-wide-auto-pruning-policy"]
+== Creating a registry-wide auto-pruning policy
+
+Registry-wide auto-pruning policies can be configured on new and existing organizations. This feature saves {productname} administrators time, effort, and storage by enforcing registry-wide rules.
+
+{productname} administrators must enable this feature by updating their `config.yaml` file through the inclusion of `DEFAULT_NAMESPACE_AUTOPRUNE_POLICY` configuration field, and one of `number_of_tags` or `creation_date` methods. Currently, this feature cannot be enabled by using the v2 UI or the API. 
+
+Use the following procedure to create an auto-prune policy for your {productname} registry.
+
+.Prerequisites
+
+* You have enabled the `FEATURE_AUTO_PRUNE` feature.
+
+.Procedure
+
+. Update your `config.yaml` file to add the `DEFAULT_NAMESPACE_AUTOPRUNE_POLICY` configuration field:
+
+.. To set the policy method to remove the oldest tags by their creation date, use the `number_of_tags` method:
```

Comment:
Might want to note here it will remove the oldest tags by creation date until the number of tags given is left.